octobot_evaluators/data_manager/matrix_manager.py

```diff
@@ -208,6 +209,47 @@ async def get_nodes_clear_event(matrix_id, nodes_paths, timeout=None):
                                              timeout=timeout) for node_path in nodes_paths])
 
 
+def get_evaluations_by_evaluator(matrix_id,
```

Comment:
Don't forget to add the cython declaration 😉, can you add some tests for this new method please ?